miso-web/src/main/webapp/pages/external/externalHeader.jsp

```diff
@@ -118,7 +118,6 @@
 </table>
 
 <script type="text/javascript">
-    setBaseUrl("<c:url value="/miso/external/"/>");
```

Comment:
(That being said, the login page external webapp doesn't seem to be complaining about this line being deleted so I will approve it)